west.yml

```diff
@@ -34,7 +34,7 @@ manifest:
     - name: rimage
       repo-path: rimage
       path: sof/rimage
-      revision: ab0429fdbe563ef6abe499c69b2483e96c4762d0
+      revision: 4ce79b152e23c7b3937a09ef57cdd69d125a9214
```

Comment:
the GUID was already changed by this https://github.com/thesofproject/sof/pull/7343/commits/b37d466a69a8b8dfb0b1ef42ff164f87b005a4b8

???

Reply:
@plbossart  That was for MTL, the same change is needed in lnl.toml.